pydy/system.py

```diff
@@ -610,3 +610,42 @@ def _Kane_constant_symbols(self):
                 *self._Kane_inlist_insyms()))
         constants.remove(dynamicsymbols._t)
         return constants
+
+    def add_gravity(self, direction=None):
```

Comment:
We will include that too within a separate ``Force`` class. Gravity is for
a quick addition for something used almost every time.

On Sat, Jun 27, 2015, 00:10 Tarun Gaba <notifications@github.com> wrote:

> In pydy/system.py
> <https://github.com/pydy/pydy/pull/249#discussion_r33383455>:
>
> > @@ -610,3 +610,42 @@ def _Kane_constant_symbols(self):
> >                  *self._Kane_inlist_insyms()))
> >          constants.remove(dynamicsymbols._t)
> >          return constants
> > +
> > +    def add_gravity(self, direction=None):
>
> Instead of add_gravity, it should be more generalized to *add and apply*
> different forces, and torques(if possible) in a single generic ClassMethod
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydy/pydy/pull/249/files#r33383455>.
>